blocks/commerce-b2b-company-structure/commerce-b2b-company-structure.js

```diff
@@ -0,0 +1,15 @@
+import { render as provider } from '../../scripts/__dropins__/storefront-company-management/render.js';
```

Comment:
I think it expects paths like `@dropins/storefront-company-management/render.js`

Reply:
- Fixed to use the full paths. \
- Changed the content and block name

Reply:
@ishakhsuvarov That's right. I haven't included the dropin code because my dropin code is based on @slavvka 's branch and I don't want to commit irrelevant files.